tests/SimplifyTest.elm

```diff
@@ -16836,7 +16836,7 @@ a = Cmd.map f Cmd.none
                     |> Review.Test.run ruleWithDefaults
                     |> Review.Test.expectErrors
                         [ Review.Test.error
-                            { message = "Using Cmd.map on Cmd.none will result in Cmd.none"
+                            { message = "Using Platform.Cmd.map on Cmd.none will result in Cmd.none"
```

Comment:
I think it's clearer to talk about `Cmd.map` rather than `Platform.Cmd.map`. I don't know how many Elm users know that that is the real name of `Cmd`.

Also for this line, it's inconsistent with `Cmd.none`.